api/src/services/sheets/sheets.js

```diff
@@ -58,4 +23,13 @@ export const createSheet = async () => {
     id: currentRestaurant.id,
     input: { sheetId: sheet?.data?.id },
   })
+
+  return sheet?.data?.id
+}
+
+export const readSheet = async ({ restaurantId }) => {
```

Comment:
Let's rename this and move it to the restaurant service. It really has nothing to do with sheets at all.